src/main/java/hudson/plugins/timestamper/io/TimestampsWriter.java

```diff
@@ -34,15 +37,9 @@
 import java.security.DigestOutputStream;
 import java.security.MessageDigest;
 import java.util.Arrays;
-
+import java.util.Optional;
```

Comment:
I'm a little concerned about whether the `timestampsDigest` field in this file. I haven't yet been able to determine whether it can be serialized to disk or not. If it can, then I may need to write a `readResolve` method to convert it from the Guava `Optional` type to the Java 8 one upon deserialization. Does anyone know how I could prove or disprove this suspicion?

Reply:
I spent some more time on this today and tried serializing both `TimestampsWriter` and `TimestamperOutputStream` using a variation of the new integration test from #43. Both attempts threw `java.io.NotSerializableException`, so I feel certain that I don't need to worry about writing a `readResolve()` method.